### PR TITLE
[tests-only] Adds e2e tests for enabling and setting quota of spaces with batch-action

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR does the following
- extends the tests to be able to enable spaces with batch actions
- extends the tests to be able to set quota with batch actions
- makes code DRY

## Related Issue

- Part of: https://github.com/owncloud/web/issues/8601

## How Has This Been Tested?

-  CI
- e2e

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/e2e/cucumber/steps/ui/adminSettings.ts [hunk 1]
```diff
@@ -89,7 +89,30 @@ When(
...
-  /^"([^"]*)" (disables|deletes) the following space(s)? using the batch-actions$/,
+  /^"([^"]*)" (changes|updates) quota of the following space(s)? to "([^"]*)" using the batch-actions$/,
+  async function (
+    this: World,
+    stepUser: string,
+    _: string,
+    __: string,
+    value: string,
+    stepTable: DataTable
+  ): Promise<void> {
```
Comment (saw-jan) [inline tests/e2e/cucumber/steps/ui/adminSettings.ts:100]:
```suggestion
  /^"([^"]*)" (?:changes|updates) quota of the following space(?:s)? to "([^"]*)" using the batch-actions$/,
  async function (
    this: World,
    stepUser: string,
    value: string,
    stepTable: DataTable
  ): Promise<void> {
```

File: tests/e2e/support/objects/app-admin-settings/spaces/actions.ts [hunk 2]
```diff
@@ -30,30 +25,68 @@ export const getDisplayedSpaces = async (page): Promise<string[]> => {
...
 }
 
+const clickOnContextMenuActionButton = async (args: {
+  page: Page
+  id: string
+  action: string
+  isBatchActions: boolean
+}): Promise<void> => {
+  const { page, id, action, isBatchActions } = args
+  let context = `.batch-actions`
```
Comment (SagarGi) [inline tests/e2e/support/objects/app-admin-settings/spaces/actions.ts:35]:
```suggestion
  let context = '.batch-actions'
```

File: tests/e2e/support/objects/app-admin-settings/spaces/actions.ts [hunk 3]
```diff
@@ -30,30 +25,68 @@ export const getDisplayedSpaces = async (page): Promise<string[]> => {
...
+  if (!isBatchActions) {
+    context = `.context-menu`
```
Comment (SagarGi) [inline tests/e2e/support/objects/app-admin-settings/spaces/actions.ts:37]:
```suggestion
    context = '.context-menu'
```

**Review by SagarGi — COMMENTED** [review]
simple changes.

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by ScharfViktor — APPROVED** [review]
LGTM

**Pull request merged**